docs/test-suite/installation-configuration.md

```diff
@@ -8,10 +8,22 @@ This page covers the requirements, installation and configuration of the [EESSI
 
 The EESSI test suite requires 
 
-* Python >= 3.6 
+* Python >= 3.7
 * [ReFrame](https://reframe-hpc.readthedocs.io) v4.3.3 (or newer)
 * [ReFrame test library (`hpctestlib`)](https://reframe-hpc.readthedocs.io/en/stable/hpctestlib.html)
 
+??? note "(If your system Python version is lower than the minimum required version, click here for some tips)
+
+    * You can upgrade and manage python versions using `pyenv`. [Link](https://github.com/pyenv/pyenv?tab=readme-ov-file#installation)
+        * For installation follow the documentation in the repository using the link above.
+        * Then follow the steps to install a new Python version and further change local paths to the new Python:
+
+                pyenv install <python version number>
+                pyenv local <python version number>
+
+    * You can install a more recent version of Python on top of the GCC/GCCcore compiler.
```

Comment:
don't forget to set `RFM_PURGE_ENVIRONMENT=1` is you use Python from a module. the ReFrame easyconfigs already do that for you.

Reply:
I do not see that env variable [here](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/r/ReFrame/ReFrame-4.6.2.eb) nor in my module env of ReFrame which was installed using EB.

Reply:
that's because your easyconfig uses the system python.

see here for an example that uses another python: https://github.com/easybuilders/easybuild-easyconfigs/blob/0835a1a062a1bbc8777a5a4164dc2e3fbf6d907b/easybuild/easyconfigs/r/ReFrame/ReFrame-4.6.3-GCCcore-13.3.0.eb#L71

Reply:
aah ok. Will change this.